modules/user/client-react/containers/ActivateUser.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { graphql } from 'react-apollo';
+
+import ActivateUser from '../components/ActivateUser';
+import ACTIVATE_USER from '../graphql/ActivateUser.graphql';
+
+class Activate extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.activate();
+  }
+
+  activate = () => {
+    try {
+      const { history, navigation } = this.props;
+      if (history) {
+        const token = this.props.match.params.token;
+        this.props.activateUser(token, history);
+      } else {
+        const { url } = navigation.state.params;
+        const [, token] = url.split('/confirmation/');
+        this.props.activateUser(token, navigation);
```

Comment:
You can get `activateUser` via destructuring

Reply:
Done